src/shared/Microsoft.Git.CredentialManager/Constants.cs

```diff
@@ -17,6 +17,10 @@ public static class EnvironmentVariables
             public const string GcmTraceMsAuth     = "GCM_TRACE_MSAUTH";
             public const string GcmDebug           = "GCM_DEBUG";
             public const string GitTerminalPrompts = "GIT_TERMINAL_PROMPT";
+            public const string CurlAllProxy       = "ALL_PROXY";
+            public const string CurlHttpProxy      = "HTTP_PROXY";
+            public const string CurlHttpsProxy     = "HTTPS_PROXY";
+            public const string GcmHttpProxy       = "GCM_HTTP_PROXY";
```

Comment:
Should Gcm have a specific Proxy Env variable?  Should it come from git instead?

Reply:
This is just here for compatibility purposes. We warn about using this deprecated option.